db/migration/migration.go

```diff
@@ -78,30 +79,67 @@ func Init() error {
 
 	logger.Info("migration path: %q, version: %d", migrationsPath, v)
 
-	return nil
+	if err = goose.LoadMigrationPlugins(migrationsPath); err != nil {
+		return fmt.Errorf("migration: failed to load migration plugins: %s", err)
+	}
+
+	if noInit {
+		// pending migrations are not allowed if no-init is enabled (no_init=true)
+		last, err := goose.LastMigration(migrationsPath)
+
+		if err != nil {
+			if err != goose.ErrNoNextVersion {
+				return fmt.Errorf("migration: %s", err)
```

Comment:
You're right - thanks. It crashed when there were no migrations at all. Fixed.
Regarding the tests - I've no idea for a simple test for that. Any suggestions?